contracts/modules/LoanOpenings.sol

```diff
@@ -360,50 +363,53 @@ contract LoanOpenings is
             _initializeInterest(
                 loanParamsLocal,
                 loanLocal,
-                sentValues[0], /// newRate
-                sentValues[1], /// newPrincipal,
-                sentValues[2] /// torqueInterest
+                sentValues.interestRate, /// newRate
+                sentValues.newPrincipal, /// newPrincipal,
+                sentValues.interestInitialAmount /// torqueInterest
             );
 
         /// substract out interest from usable loanToken sent.
-        sentValues[3] = sentValues[3].sub(amount);
+        sentValues.loanTokenSent = sentValues.loanTokenSent.sub(amount);
 
         if (isTorqueLoan) {
-            require(sentValues[3] == 0, "surplus loan token");
+            require(sentValues.loanTokenSent == 0, "surplus loan token");
 
-            uint256 borrowingFee = _getBorrowingFee(sentValues[4]);
+            uint256 borrowingFee = _getBorrowingFee(sentValues.collateralTokenSent);
             // need to temp into local state to avoid
             address _collateralToken = loanParamsLocal.collateralToken;
             address _loanToken = loanParamsLocal.loanToken;
             if (borrowingFee != 0) {
                 _payBorrowingFee(
-                    sentAddresses[1], /// borrower
+                    sentAddresses.borrower, /// borrower
                     loanLocal.id,
                     _collateralToken, /// fee token
                     _loanToken, /// pairToken (used to check if there is any special rebates or not) -- to pay fee reward
                     borrowingFee
                 );
 
-                sentValues[4] = sentValues[4] /// collateralTokenReceived
-                    .sub(borrowingFee);
+                sentValues.collateralTokenSent = sentValues.collateralTokenSent.sub(borrowingFee);
```

Comment:
adding `collateralTokenReceived` to the struct will make the code much clearer
```
sentValues.collateralTokenReceived = sentValues.collateralTokenSent.sub(borrowingFee);
```
which also requires minor code changes in this function to use collateralTokenReceived as needed but will increase readability

Reply:
We have discussed this, it's just a matter of inaccurrate comment. So adding new variable to the struct for the same purpose is not really good idea